tests/event_wait_timeout/Makefile

```diff
@@ -3,6 +3,7 @@ include ../Makefile.tests_common
 FORCE_ASSERTS = 1
 USEMODULE += event
 USEMODULE += xtimer
+USEMODULE += ztimer ztimer_usec
```

Comment:
This change means that `xtimer_on_ztimer` is pulled in so the whole test now runs on `ztimer`, not sure how to proceed here, in a lot of cases `ztimer` is not tested unless explicitly pulled in, but in this case, maybe the best is having an individual test?